xrdpdev/xrdpdev.c

```diff
@@ -723,7 +765,8 @@ rdpScreenInit(ScreenPtr pScreen, int argc, char **argv)
 
     RegisterBlockAndWakeupHandlers(rdpBlockHandler1, rdpWakeupHandler1, pScreen);
 
-    g_timer = TimerSet(g_timer, 0, 10, rdpDeferredRandR, pScreen);
+    g_randr_timer = TimerSet(g_randr_timer, 0, 10, rdpDeferredRandR, pScreen);
+    g_damage_timer = TimerSet(g_damage_timer, 0, 10, rdpDeferredDamage, pScreen);
```

Comment:
Why the 10ms timer here? Is this repeating or just a hack around something not being initialized yet, which could break if 10ms isn't long enough. Maybe add a comment?

Reply:
I think they are initialized just after the current call.  The 10 can be anything bigger then 0.  If I remember right, 0 does nothing.  Let me see if there is another way to register these things at the right point.